src/components.js

```diff
@@ -3,28 +3,45 @@ import * as React from 'react'
 export class GameView extends React.Component {
 
   checkGameStatus() {
-    return (this.props.game.isGameOver() )
+    return ( this.props.game.game1.isGameOver() || this.props.game.game2.isGameOver() )
   }
 
   render() {
-    const classString = this.props.game.fallingPiece.classString
-    console.log(this.props)
     return (
-      <div className='border' style={{ width: this.props.game.cols*25,
-        height: this.props.game.rows*25}}>
+      <div>
+        <NextPieceView piece={this.props.game.game1.nextPiece} className='next1'/>
+      <div className='border game-1' style={{ width: this.props.game.game1.cols*25,
+        height: this.props.game.game1.rows*25}}>
+        { this.checkGameStatus() ?
+          <span><h2 className="game-over">GAME OVER</h2>
+            <PieceView piece={this.props.game.game1.fallingPiece} />
+            <RubbleView rubble={this.props.game.game1.rubble} />
+            <ScoreView score={this.props.game.game1.score} />
+          </span> :
+          <span>
+            <PieceView piece={this.props.game.game1.fallingPiece} />
+            <RubbleView rubble={this.props.game.game1.rubble} />
+            <ScoreView score={this.props.game.game1.score} />
+          </span>
```

Comment:
Related to the note above about passing in only what is needed - it seems as if these `span`s are really another component - the only difference between this set of `span`s, and the set below, is the *data*.  With some refactoring, we could do something like this (from wherever the `GameView` is currently being created) - grab me if you want to discuss this:
```jsx
<GameView rows={rows} cols={cols}>
  <Game {...game1} />
  <Game {...game2} />
</GameView>
```